<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-03T06:58:59 IST" name="Suite" finished-at="2023-06-03T07:00:38 IST" duration-ms="99065">
    <groups>
      <group name="Master">
        <method signature="TC_001_AcctRegistration.test_account_registration()[pri:0, instance:testCases.TC_001_AcctRegistration@6379eb]" name="test_account_registration" class="testCases.TC_001_AcctRegistration"/>
        <method signature="TC_002_LoginTest.login_test()[pri:0, instance:testCases.TC_002_LoginTest@7ec7ffd3]" name="login_test" class="testCases.TC_002_LoginTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC_002_LoginTest.login_test()[pri:0, instance:testCases.TC_002_LoginTest@7ec7ffd3]" name="login_test" class="testCases.TC_002_LoginTest"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="TC_001_AcctRegistration.test_account_registration()[pri:0, instance:testCases.TC_001_AcctRegistration@6379eb]" name="test_account_registration" class="testCases.TC_001_AcctRegistration"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2023-06-03T06:58:59 IST" name="Test" finished-at="2023-06-03T07:00:38 IST" duration-ms="99065">
      <class name="testCases.TC_002_LoginTest">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:testCases.TC_002_LoginTest@7ec7ffd3]" started-at="2023-06-03T07:00:03 IST" name="setUp" finished-at="2023-06-03T07:00:21 IST" duration-ms="18088" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login_test()[pri:0, instance:testCases.TC_002_LoginTest@7ec7ffd3]" started-at="2023-06-03T07:00:21 IST" name="login_test" finished-at="2023-06-03T07:00:37 IST" duration-ms="16201" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Menu My Account not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Menu My Account not found
at org.testng.Assert.fail(Assert.java:99)
at testCases.TC_002_LoginTest.login_test(TC_002_LoginTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_test -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_002_LoginTest@7ec7ffd3]" started-at="2023-06-03T07:00:38 IST" name="tearDown" finished-at="2023-06-03T07:00:38 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_002_LoginTest -->
      <class name="testCases.TC_001_AcctRegistration">
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:testCases.TC_001_AcctRegistration@6379eb]" started-at="2023-06-03T06:59:00 IST" name="setUp" finished-at="2023-06-03T06:59:43 IST" duration-ms="42492" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="test_account_registration()[pri:0, instance:testCases.TC_001_AcctRegistration@6379eb]" started-at="2023-06-03T06:59:43 IST" name="test_account_registration" finished-at="2023-06-03T07:00:01 IST" duration-ms="18093" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test failed
at org.testng.Assert.fail(Assert.java:99)
at testCases.TC_001_AcctRegistration.test_account_registration(TC_001_AcctRegistration.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_account_registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_001_AcctRegistration@6379eb]" started-at="2023-06-03T07:00:03 IST" name="tearDown" finished-at="2023-06-03T07:00:03 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_001_AcctRegistration -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
